Sources/PointFree/Account/Account.swift

```diff
@@ -9,69 +9,11 @@ import Prelude
 import Styleguide
 import Tuple
 
-public struct Flash: Codable {
-  public enum Priority: String, Codable {
-    case error
-    case notice
-    case warning
-  }
-
-  public let priority: Priority
-  public let message: String
-}
-
-struct SimplePageLayoutData<A> {
-  let currentUser: Database.User?
-  let data: A
-  let flash: [Flash.Priority: [Node]]
-  let title: String
-}
-
-func simplePageLayout<A>(_ contentView: View<A>) -> View<SimplePageLayoutData<A>> {
-  return View { layoutData in
-    document([
-      html([
-        head([
-          title(layoutData.title),
-          style(renderedNormalizeCss),
-          style(styleguide),
-          style(render(config: pretty, css: pricingExtraStyles)),
-          meta(viewport: .width(.deviceWidth), .initialScale(1)),
-          ]),
-        body(
-          darkNavView.view((layoutData.currentUser, nil))
-            <> [
-              gridRow([
-                gridColumn(sizes: [.mobile: 12, .desktop: 8], [style(margin(leftRight: .auto))], [
-                  div(
-                    [`class`([Class.padding([.mobile: [.all: 3], .desktop: [.all: 4]])])],
-                    contentView.view(layoutData.data)
-                  )
-                  ])
-                ])
-            ]
-            <> footerView.view(unit)
-        )
-        ])
-      ])
-  }
-}
-
-let wrappedAccountView = simplePageLayout(accountView)
-  .contramap { subscription, teamInvites, teammates, currentUser in
-    SimplePageLayoutData(
-      currentUser: currentUser,
-      data: (subscription, teamInvites, teammates, currentUser),
-      flash: [:],
-      title: "Account"
-    )
-}
-
 let accountResponse =
   filterMap(require1 >>> pure, or: loginAndRedirect)
     <| fetchAccountData
     >-> writeStatus(.ok)
-    >-> respond(wrappedAccountView.contramap(lower))
+    >-> respond(accountView.contramap(lower), layout: simplePageLayout(title: "Account", currentUser: get4))
```

Comment:
This is what rendering with layout looks like. May change to better fit our needs in the future!

Reply:
The latter was the thing I tried first but it still felt heavier than this layout-specific solution. Wanna sketch something out?

Reply:
(Layout data via contra map still needs to pluck out current user and pass along the view data intact, so I made this helper contain that logic and forward the one shared but instead: current user.)